js/ui/overlay.d.ts

```diff
@@ -67,6 +67,14 @@ export interface dxOverlayOptions<T = dxOverlay> extends WidgetOptions<T> {
      * @public
      */
     dragEnabled?: boolean;
+    /**
+     * @docid
+     * @deprecated
+     * @default {}
+     * @prevFileNamespace DevExpress.core
+     * @public
+     */
+     elementAttr?: any;
```

Comment:
there is one excess space on the row beginning

Reply:
done